.gitignore

```diff
@@ -19,3 +19,4 @@ npm-debug.log
 
 node_modules
 coverage
+doc/annotated
```

Comment:
in package.json it is `"annotate": "docco redis-store.js -o docs/annotated"`
Either this or the package needs to be changed